### DISCUSSION TITLE
QUploaderProps onFailed

### DISCUSSION THREAD

**Discussion by maggie44** [quasarframework/quasar#12676] [General - Components / Directives / etc] [answered]
I feel I must be doing something stupid here. I am using the onFailed via the QUploader in Typescript. Declaring using QUploaderProps['onFailed'] but it is showing an error. 

<img width="753" alt="Screenshot 2022-03-06 at 11 09 21" src="https://user-images.githubusercontent.com/64841595/156920531-0707e8d8-24a2-407d-9261-8fdc238a98c4.png">

```
    function onUploaderFailed(info: QUploaderProps['onFailed']) {
      console.log(info)
      $q.notify({
        type: 'negative',
        message: t('general.Error')
      })
    }
```

**Comment by yusufkandemir** [comment, marked as answer]
`QUploaderProps['onFailed']` is the function itself, but you are structuring it in a way that is as if it's the parameter. You would need to do this instead:

```ts
const onUploaderFailed: QUploaderProps['onFailed'] = (info) => {
  console.log(info)
  $q.notify({
    type: 'negative',
    message: t('general.Error')
  })
}
```
